chapters/syntax.tex

```diff
@@ -18,16 +18,17 @@ \section{Lexical conventions}\label{lexical-conventions}
 NONDIGIT = "_" | letters "a" to "z" | letters "A" to "Z"
 STRING = """ { S-CHAR | S-ESCAPE } """
 S-CHAR = see below
-Q-CHAR = NONDIGIT | DIGIT | "!" | "#" | "$" | "%" | "&" | "(" | ")" | "*" | "+" | "," |
-         "-" | "." | "/" | ":" | ";" | "<" | ">" | "=" | "?" | "@" | "[" | "]" | "^" |
-        "{" | "}" | "|" | "~" | " " | """
+Q-CHAR = NONDIGIT | DIGIT | "!" | "#" | "$" | "%" | "&" | "(" | ")" 
+   | "*" | "+" | "," | "-" | "." | "/" | ":" | ";" | "<" | ">" | "=" 
+   | "?" | "@" | "[" | "]" | "^" | "{" | "}" | "|" | "~" | " " | """
 S-ESCAPE = "\'" | "\"" | "\?" | "\\" |
          "\a" | "\b" | "\f" | "\n" | "\r" | "\t" | "\v"
 DIGIT = "0" | "1" | "2" | "3" | "4" | "5" | "6" | "7" | "8" | "9"
 UNSIGNED-INTEGER = DIGIT { DIGIT }
 UNSIGNED-REAL = UNSIGNED-INTEGER  "." [ UNSIGNED-INTEGER ]
-     |  UNSIGNED_INTEGER [ "." [ UNSIGNED_INTEGER ] ] ( "e" | "E" ) [ "+" | "-" ] UNSIGNED-INTEGER
-     |   "."  UNSIGNED-INTEGER [ ( "e" | "E" ) [ "+" | "-" ] UNSIGNED-INTEGER ]
+   |  UNSIGNED_INTEGER [ "." [ UNSIGNED_INTEGER ] ] 
+        ( "e" | "E" ) [ "+" | "-" ] UNSIGNED-INTEGER
+   |   "."  UNSIGNED-INTEGER [ ( "e" | "E" ) [ "+" | "-" ] UNSIGNED-INTEGER ]
 \end{lstlisting}
 \textrm{S-CHAR} is any member of the Unicode character set
 (\url{http://www.unicode.org}; see \cref{mapping-package-class-structures-to-a-hierarchical-file-system} for storing as UTF-8 on files) except double-quote """, and backslash "\textbackslash{}"
```

Comment:
When we're at it, shouldn't we fix this bad use of un-paired double quotation marks (and end a sentence with period)?
```suggestion
(\url{http://www.unicode.org}; see \cref{mapping-package-class-structures-to-a-hierarchical-file-system} for storing as UTF-8 on files) except double-quote `"', and backslash `\textbackslash{}'.
```